stan/math/opencl/kernels/normal_id_glm_lpdf.hpp

```diff
@@ -0,0 +1,162 @@
+#ifndef STAN_MATH_OPENCL_KERNELS_NORMAL_ID_GLM_LPDF_HPP
+#define STAN_MATH_OPENCL_KERNELS_NORMAL_ID_GLM_LPDF_HPP
+#ifdef STAN_OPENCL
+
+#include <stan/math/opencl/kernel_cl.hpp>
+
+namespace stan {
+namespace math {
+namespace opencl_kernels {
+
+// \cond
+static const char* normal_id_glm_kernel_code = STRINGIFY(
+    // \endcond
+    /**
+     * GPU implementation of Generalized Linear Model (GLM)
+     * with Normal distribution and identity link function.
+     *
+     * Must be run with at least N threads and local size equal to LOCAL_SIZE_.
+     * @param[in] y_glob vector parameter
```

Comment:
glob?

Reply:
Short for global. I can change the variable name if you want.

Reply:
tbh I think it's fine to have y here

Reply:
Right. I thing I copied this from some other GLM that had y in local variable too.